build/yamls/base/controller.yml

```diff
@@ -2,6 +2,10 @@
 apiVersion: v1
 kind: Service
 metadata:
+  annotations:
+    prometheus.io/port: "443"
```

Comment:
Might be helpful to add some comments to explain why these annotations are required.

Reply:
Found out that these annotations are only useful for pods, but not for services and daemonsets.
I will remove these.